tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_graphics_GC.java

```diff
@@ -824,9 +825,7 @@ public void test_bug1288_createGCFromImageFromNonDisplayThread() throws Interrup
 	AtomicReference<Exception> exceptionReference = new AtomicReference<>();
 	Thread thread = new Thread(() -> {
 		try {
-			Image image = new Image(null, 100, 100);
```

Comment:
This change is plain wrong as it makes the test meaningless! Please review each case where you change tests as there are a number of such cases.